visualization/src/resource-visualization.cc

```diff
@@ -256,31 +306,55 @@ void visualizeReprojectedDepthResource(
           return;
         }
 
+        if (FLAGS_vis_pointcloud_filter_dense_map_before_publishing) {
```

Comment:
Overall I can't figure that this if statement does anything, should this be removed completely?